src/bundle.ts

```diff
@@ -83,14 +122,16 @@ export class BuildBundler extends Transform {
   }
 
   async _buildBundles(): Promise<Map<string, string>> {
-    let strategy: BundleStrategy;
-    if (this.config.shell) {
+    let strategy = this._strategy;
```

Comment:
No need to change, but... :)

Ternaries are a little better because they avoid boolean conversion, and sometimes the logic is easier to follow than compound logical expressions:

```ts
const strategy = this._strategy !== undefined
    ? this._trategy
    : this.config.shell !== undefined
        ? generateShellMergeStrategy(urlFromPath(this.config.root, this.config.shell));
        : undefined;
```

I can read this as if/else/if/else...